### PR TITLE
AlmaLinux/build-system#67: Create a public documentation and related repositories for building AL by a simple user

### PR DESCRIPTION
**Pull request by soksanichenko**

### CODE REVIEW

File: docs/development/How to build AL distribution.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* 300+ GB free storage space
+
+## Deploying a Pungi node
+
+That Ansible scenario prepare your machine to building a distribution
+
+1. Install Docker CE and prepare docker containers to build the distributions inside them
+2. Setup systemd timer to sync a local mirror of packages. They will be used for building
+
+## Do the following step to prepare your machine
```
Comment (anfimovdm) [inline docs/development/How to build AL distribution.md:21]: steps

**Review by anfimovdm — APPROVED** [review]

File: docs/development/How to build AL distribution.md (hunk 1 above)
Comment (soksanichenko) [reply docs/development/How to build AL distribution.md]: Done

File: docs/development/How to build AL distribution.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: 'AL Building'
+---
+
+# How to build AlmaLinux OS distribution on one's own
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:5]: How about calling it "Building AlmaLinux OS locally"?

File: docs/development/How to build AL distribution.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Hardware and software requirements to a final machine
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:7]: "Hardware and software requirements" is enough I think

File: docs/development/How to build AL distribution.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+# How to build AlmaLinux OS distribution on one's own
+
+## Hardware and software requirements to a final machine 
+
+* AlmaLinux 9
+* 6+ CPU cores
+* 8+ GB RAM
+* 300+ GB free storage space
+
+## Deploying a Pungi node
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:14]: I'd probably merge the two sections "Deploying a Pungi node" and "Do the following steps to prepare your machine" into a single one, "Setting up your machine" or something like that. Then, explain why we need a pungi node, and finally, put the step by step process as you already have.

File: docs/development/How to build AL distribution.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+That Ansible scenario prepare your machine to building a distribution
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:16]: Maybe, something like:
"""
The following process will guide you through the process of:
1. Installing Docker CE and prepare docker containers to build the distributions inside them
2. Setting up a __systemd timer__ to synchronize a local mirror of packages, which will be used later for building the distribution
"""

File: docs/development/How to build AL distribution.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Install Docker CE and prepare docker containers to build the distributions inside them
+2. Setup systemd timer to sync a local mirror of packages. They will be used for building
+
+## Do the following steps to prepare your machine
+
+1. Clone GitHub repository [pungi-node-public](https://github.com/AlmaLinux/pungi-node-public)
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:23]: Clone this Github repository:

File: docs/development/How to build AL distribution.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+That Ansible scenario prepare your machine to building a distribution
+
+1. Install Docker CE and prepare docker containers to build the distributions inside them
+2. Setup systemd timer to sync a local mirror of packages. They will be used for building
+
+## Do the following steps to prepare your machine
+
+1. Clone GitHub repository [pungi-node-public](https://github.com/AlmaLinux/pungi-node-public) 
+2. Prepare Python virtual environment if you don't have installed Ansible
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:24]: .. if you don't have Ansible installed

File: docs/development/How to build AL distribution.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+   1. Run command `python3 -m venv ./venv` in a suitable for you directory
+   2. Activate venv by command `source ./venv/bin/activate`
+   3. Install Ansible by command `pip install ansible`
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:27]: ditto

File: docs/development/How to build AL distribution.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Install Docker CE and prepare docker containers to build the distributions inside them
+2. Setup systemd timer to sync a local mirror of packages. They will be used for building
+
+## Do the following steps to prepare your machine
+
+1. Clone GitHub repository [pungi-node-public](https://github.com/AlmaLinux/pungi-node-public) 
+2. Prepare Python virtual environment if you don't have installed Ansible
+   1. Run command `python3 -m venv ./venv` in a suitable for you directory
+   2. Activate venv by command `source ./venv/bin/activate`
```
[the remark this answers is not in the document]
Comment (javihernandez) [reply docs/development/How to build AL distribution.md:26]: or "Activate the virtual environment running the command ..."

File: docs/development/How to build AL distribution.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+   3. Install Ansible by command `pip install ansible`
+3. Copy `inventories/sample` as a new Ansible inventory which should fill
```
Comment (javihernandez) [inline docs/development/How to build AL distribution.md:28]: ... as the new Ansible inventory you need to fill out ?

**Review by anfimovdm — APPROVED** [review]

**Review by Korulag — APPROVED** [review]

**Review by javihernandez — APPROVED** [review]
LGTM. Can you please squash before merge?

**Comment by soksanichenko** [comment]
I'll squash. Thanks.

**Pull request merged**